erpnext/accounts/report/cost_center_financial_statements/cost_center_financial_statements.py

```diff
@@ -21,7 +21,7 @@ def execute(filters=None):
 	if not filters.get('cost_center'):
 		frappe.msgprint(_("Please select at least one cost center."));
 
-	cost_centers = get_cost_centers_with_children(filters.cost_center)
+	cost_centers = filters.cost_center
```

Comment:
- I suggested to add a check in the reports UI as `Include child Cost Centers`.
- If that checkbox is not checked, the dont run this function `get_cost_centers_with_children`, else we need that function to run